libbeat/docs/shared-autodiscover.asciidoc

```diff
@@ -178,8 +178,8 @@ Configuration parameters:
  * `node` or `namespace`: Specify labels and annotations filters for the extra metadata coming from node and namespace. By default all labels are included while annotations are not. To change default behaviour `include_labels`, `exclude_labels` and `include_annotations` can be defined. Those settings are useful when storing labels and annotations that require special handling to avoid overloading the storage output.
  Note: wildcards are not supported for those settings.
  The enrichment of `node` or `namespace` metadata can be individually disabled by setting `enabled: false`.
- * `deployment`: If resource is `pod` and it is created from a `deployment`, by default the deployment name is added, this can be disabled by setting `deployment: false`.
- * `cronjob`: If resource is `pod` and it is created from a `cronjob`, by default the cronjob name is added, this can be disabled by setting `cronjob: false`.
+ * `deployment`: If resource is `pod` and it is created from a `deployment`, by default the deployment name is not added, this can be enabled by setting `deployment: true`.
+ * `cronjob`: If resource is `pod` and it is created from a `cronjob`, by default the cronjob name is not added, this can be enabled by setting `cronjob: true`.
```

Comment:
```suggestion
 * `cronjob`: If resource is `pod` and it is created from a `cronjob`. By default the cronjob name is not added, this can be enabled by setting `cronjob: true`.
```